pkgs/by-name/dr/dronecan-gui-tool/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  libsForQt5,
+  imagemagick,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "dronecan-gui-tool";
+  version = "1.2.25";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "dronecan";
+    repo = "gui_tool";
+    rev = "v${version}";
+    sha256 = "sha256-PtiCQTaGXfqIkGlGdiWcRxxG0CUshEr/foeqwqJa7DE=";
+  };
+
+  build-system = with python3Packages; [
+    install-freedesktop
+    setuptools-git
+    wheel
+    libsForQt5.wrapQtAppsHook
+    imagemagick
+  ];
+
+  dependencies = with python3Packages; [
+    pkgs.iproute2
+    dronecan
+    easywebdav
+    ipykernel
+    jupyter-client
+    numpy
+    pygments
+    pymavlink
+    pymonocypher
+    pyqt5
+    pyqtgraph
+    pyserial
+    pyyaml
+    qtawesome
+    qtconsole
+    qtpy
+    qtwidgets
+    setuptools
+    traitlets
+  ];
+
+  dontWrapQtApps = true;
+
+  postInstall = ''
+    convert icons/logo.ico icons/dronecan_gui_tool.png
+    mv $out/share/applications/DroneCAN\ GUI\ Tool.desktop $out/share/applications/dronecan_gui_tool.desktop
+    substituteInPlace $out/share/applications/dronecan_gui_tool.desktop \
+      --replace "Exec=/dronecan_gui_tool-1.2.25.data/scripts/dronecan_gui_tool" "Exec=dronecan_gui_tool"
```

Comment:
Don't include the version `1.2.25` verbatim. Maybe use `{version}` or replace it using regex.